src/main/java/hudson/plugins/git/GitStatus.java

```diff
@@ -324,7 +324,7 @@ public void writeBody(PrintWriter w) {
         @SuppressFBWarnings(value="NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE", justification="Jenkins.getInstance() is not null")
         public List<ResponseContributor> onNotifyCommit(String origin, URIish uri, String sha1, List<ParameterValue> buildParameters, String... branches) {
             if (LOGGER.isLoggable(Level.FINE)) {
-                LOGGER.fine("Received notification from " + StringUtils.defaultIfBlank(origin, "?")
+                LOGGER.fine("Received notification from " + StringUtils.defaultIfEmpty(origin, "?")
```

Comment:
The project compiles fine. Looks like I had some local synchronization issues. Dropped the commit.